### ISSUE TITLE
Clear output views (results and status) when running a query

### ISSUE THREAD

**Issue by beRto-** [sqlitebrowser/sqlitebrowser#2568]
Describe the new feature
--------------------------
**TLDR - request to clear previous / stale data from the various output windows as soon as I run a query. Repopulate the windows with new data only when the query completes.**

When I run a query ("QUERY1"), I get a results output and a status output (# rows returned in XX seconds, etc.). This is great.

I have noticed that if I then run a second query ("QUERY2", the status and results from QUERY1 continue to show in the output views until QUERY2 has completed.

This is not a problem normally, where a query takes a few ms. But I find this confusing when QUERY2 takes more than a couple of seconds because the stale QUERY1 outputs make it *look* like QUERY2 has completed.

I need to keep an eye out in the bottom right corner for the "red x" and the "Busy (reading rows)" status messages to let me know when the query completes. It would be cleaner if the results windows were cleared as soon as I run QUERY2, and then populated once QUERY2 completes.

Additional info
---------------
Please answer these questions before submitting your feature request.

#### Is your feature request related to an issue? Please include the issue number.
NO. This is a general feature / user expectations request.

#### Does this feature exist in another product or project? Please provide a link.
YES. I think all other DB management tools (e.g. Aginity Workbench, MS SQL Server Management Studio) I have used clear the output views as soon as a new query is executed.

#### Do you have a screenshot? Please add screenshots to help explain your idea.
Sorry, I do not have these at hand. But if the description is not clear, I can set something up to demonstrate and screen capture.
